src/components/Settings/LLMSettings.tsx

```diff
@@ -136,12 +147,9 @@ const LLMSettings: React.FC<LLMSettingsProps> = ({
             <div>
               <h4 className="text-gray-100 mb-1">Default LLM:</h4>
               <div className="w-full mb-1">
-                <CustomSelect
-                  isLLMDropdown={true}
-                  options={modelOptions}
-                  value={defaultModel}
-                  onChange={handleDefaultModelChange}
-                  onDelete={handleDeleteModel}
```

Comment:
Why did we delete the ability to remove models?

Reply:
Probably in simplifying the chat, we dont want to keep supporting this feature. Not sure if we would need to bring it back, but OTOH, doesn't feel like an important thing to focus on